fsw/src/sc_cmds.c

```diff
@@ -543,11 +541,11 @@ void SC_WakeupCmd(const SC_WakeupCmd_t *Cmd)
     /*
      * Time to execute a command in the SC memory
      */
-    while (SC_OperData.NumCmdsSec < SC_MAX_CMDS_PER_WAKEUP)
+    while (SC_OperData.NumCmdsWakeup < SC_MAX_CMDS_PER_WAKEUP)
     {
-        SC_UpdateNextTime();
+        SC_GetNextRtsTime();
```

Comment:
I recommend moving the `SC_GetNextRtsTime` call down to line 562, right before the `SC_ProcessRtpCommand` call since you only need to get this information if deciding to send from the RTS.